sympy/solvers/ode/tests/test_ode.py

```diff
@@ -3701,74 +3697,69 @@ def test_sysode_linear_neq_order1():
-    sol10 = [Eq(x(t), C1*exp(2*t) + C2), Eq(y(t), C1 * exp(2*t) - C2)]
+    sol10 = [Eq(x(t), -C1 + C2*exp(2*t)), Eq(y(t), C1 + C2*exp(2*t))]
     assert dsolve(eq10) == sol10
     assert checksysodesol(eq10, sol10) == (True, [0, 0])
 
     eq11 = [Eq(x(t).diff(t), 2*x(t) + y(t)), Eq(y(t).diff(t), -x(t) + 2*y(t))]
-    sol11 = [Eq(x(t), (C1*cos(t) + C2*sin(t)) * exp(2*t)),
-             Eq(y(t), (-C1*sin(t) + C2*cos(t)) * exp(2*t))]
+    sol11 = [Eq(x(t), C1*exp(2*t)*sin(t) + C2*exp(2*t)*cos(t)),
+            Eq(y(t), C1*exp(2*t)*cos(t) - C2*exp(2*t)*sin(t))]
     assert dsolve(eq11) == sol11
     assert checksysodesol(eq11, sol11) == (True, [0, 0])
 
     eq12 = [Eq(x(t).diff(t), x(t) + 2*y(t)), Eq(y(t).diff(t), 2*x(t) + y(t))]
-    #sol12 = [Eq(x(t), C1*exp(-t) + C2*exp(3*t)),
-    #         Eq(y(t), -C1*exp(-t) + C2*exp(3*t))]
-    sol12 = [Eq(x(t), 2*C2*exp(-t) + 2*C1*exp(3*t)),
-             Eq(y(t), -2*C2*exp(-t) + 2*C1*exp(3*t))]
+    sol12 = [Eq(x(t), -C1*exp(-t) + C2*exp(3*t)), Eq(y(t), C1*exp(-t) + C2*exp(3*t))]
     assert dsolve(eq12) == sol12
     assert checksysodesol(eq12, sol12) == (True, [0, 0])
 
     eq13 = [Eq(x(t).diff(t), 4*x(t) + y(t)), Eq(y(t).diff(t), -x(t) + 2*y(t))]
-    sol13 = [Eq(x(t), (C1 + C2*(t+1)) * exp(3*t)),
-             Eq(y(t), -(C1 + C2*t) * exp(3*t))]
+    sol13 = [Eq(x(t), C1*exp(3*t) + C2*(t*exp(3*t) + exp(3*t))),
+            Eq(y(t), -C1*exp(3*t) - C2*t*exp(3*t))]
     assert dsolve(eq13) == sol13
     assert checksysodesol(eq13, sol13) == (True, [0, 0])
 
     eq14 = [Eq(x(t).diff(t), a*y(t)), Eq(y(t).diff(t), a*x(t))]
-    sol14 = [Eq(x(t), C1*exp(a*t) + C2*exp(-a*t)), Eq(y(t), C1*exp(a*t) - C2*exp(-a*t))]
+    sol14 = [Eq(x(t), -C1*exp(-a*t) + C2*exp(a*t)), Eq(y(t), C1*exp(-a*t) + C2*exp(a*t))]
     #assert dsolve(eq14) == sol14
     assert checksysodesol(eq14, sol14) == (True, [0, 0])
 
     eq15 = [Eq(x(t).diff(t), a*y(t)), Eq(y(t).diff(t), b*x(t))]
     sol15 = [Eq(x(t), -C1*a*exp(-t*sqrt(a*b))/sqrt(a*b) + C2*a*exp(t*sqrt(a*b))/sqrt(a*b)),
-             Eq(y(t), C1*exp(-t*sqrt(a*b)) + C2*exp(t*sqrt(a*b)))]
+            Eq(y(t), C1*exp(-t*sqrt(a*b)) + C2*exp(t*sqrt(a*b)))]
     #assert dsolve(eq15) == sol15
     assert checksysodesol(eq15, sol15) == (True, [0, 0])
 
     eq16 = [Eq(x(t).diff(t), a*x(t) + b*y(t)), Eq(y(t).diff(t), c*x(t))]
-    sol16 = [Eq(x(t), -2*C2*b*exp(t*(a/2 - sqrt(a**2 + 4*b*c)/2))/(a + sqrt(a**2 + 4*b*c))
-                     - 2*C3*b*exp(t*(a/2 + sqrt(a**2 + 4*b*c)/2))/(a - sqrt(a**2 + 4*b*c))),
-            Eq(y(t), C2*exp(t*(a/2 - sqrt(a**2 + 4*b*c)/2))
-                   + C3*exp(t*(a/2 + sqrt(a**2 + 4*b*c)/2)))]
+    sol16 = [Eq(x(t), -2*C1*b*exp(t*(a/2 - sqrt(a**2 + 4*b*c)/2))/(a + sqrt(a**2 + 4*b*c)) - 2*C2*b*exp(t*(a/2 + sqrt(a**2 + 4*b*c)/2))/(a - sqrt(a**2 + 4*b*c))),
+            Eq(y(t), C1*exp(t*(a/2 - sqrt(a**2 + 4*b*c)/2)) + C2*exp(t*(a/2 + sqrt(a**2 + 4*b*c)/2)))]
     #assert dsolve(eq16) == sol16
```

Comment:
Does `dsolve` return the changed solution here?

Reply:
Yes it does. As before, I will uncomment this line.

Reply:
There might be more examples of this